release-notes/kvm/v11.1.0.md

```diff
@@ -1,5 +1,7 @@
 ## :zap: Giant Swarm Release 11.1.0 for KVM is now active for you! :zap:
 
+This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.
```

Comment:
```suggestion
This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also enabled the metrics service. This allows prometheus-operator to scrape for metrics and improves our monitoring of the app.
```

Same suggestion as in AWS release PR.

Reply:
> improves our monitoring of the app

Customer (adidas) requested the feature, for them to be able to monitor nginx IC, using prometheus-operator (managed prometheus); they manage that prometheus-operator too at the moment. We may eventually end up using this metrics Service too (enabling it by default), but not yet.